bindings/bindingtester/run_binding_tester.sh

```diff
@@ -25,7 +25,8 @@ BREAKONERROR="${BREAKONERROR:-0}"
 RUNSCRIPTS="${RUNSCRIPTS:-1}"
 RUNTESTS="${RUNTESTS:-1}"
 RANDOMTEST="${RANDOMTEST:-0}"
-BINDINGTESTS="${BINDINGTESTS:-python python3 java java_async ruby go flow}"
+# BINDINGTESTS="${BINDINGTESTS:-python python3 java java_async ruby go flow}"
+BINDINGTESTS="${BINDINGTESTS:-python python3 java java_async go flow}"
```

Comment:
I think this may be the only change that we need. You'll want to keep the known_testers change to run locally, and I think run_tester_loop may just exist as a tool for running tests locally too.

Reply:
Done.